### PR TITLE
Add multiple files support.

### PR DESCRIPTION
**Pull request by gregberge**
It can be useful to target several files, I don't know if it is in the philosophy of the library.

### CODE REVIEW

File: lib/entry.js [hunk 1]
```diff
@@ -545,13 +557,14 @@ _.assign(Entry.prototype, {
...
   'getType': getType,
+  'getUrl': getUrl,
   'isAlias': isAlias,
   'isCtor': isCtor,
   'isFunction': isFunction,
   'isLicense': isLicense,
   'isPlugin': isPlugin,
   'isPrivate': isPrivate,
-  'isStatic': isStatic
+  'isStatic': isStatic,
```
Comment (jdalton) [inline lib/entry.js]: Trailing comma.

**Comment by neoziro** [comment]
@jdalton fixed

**Comment by neoziro** [comment]
@jdalton do you agree?

**Comment by jdalton** [comment]
I'll need some time to seriously mull it over. Christmas Eve is probably not the time. 🎅

**Comment by neoziro** [comment]
@jdalton take your time, Merry Christmas!🎅

**Comment by stezu** [comment]
This is also something I'm interested in for some of my projects. I was actually planning on making a pull request until I saw this one.

Is there something that concerns you about supporting multiple files?

**Comment by neoziro** [comment]
@jdalton did you take a look?

**Pull request closed without merging**